command.go

```diff
@@ -197,6 +204,12 @@ func (cmd *CommandFunc) Call(ctx context.Context, args, env []string) (int, erro
 	}
 
 	for name, field := range cmd.options {
+		if cmd.IgnoreEnvOptions != nil {
+			if _, ok := cmd.IgnoreEnvOptions[name]; ok {
+				continue
+			}
+		}
```

Comment:
Lookups in `nil` maps are valid in Go:
```suggestion
			if _, ok := cmd.IgnoreEnvOptions[name]; ok {
				continue
			}
```